### PR TITLE
Install and setup asset repo using operators

### PR DESCRIPTION
**Pull request by DanRoseus**

### CODE REVIEW

File: products/bash/ar_remote_create.sh [hunk 1]
```diff
@@ -139,14 +144,6 @@ function test_remote() {
 }
 
 ## retries request until we get a 200, or hit max retries
-echo "ar_path=$ar_path"
-echo "token=$token"
-echo "remote=$remote"
-
-
-curl -X POST --insecure -v https://$ar_path/api/remotes/test -d "$remote" -H "Content-Type: application/json" -H "Authorization: Bearer $token" -o ./ar_create_tmp/remote-status.log
```
Comment (James-Kirk) [inline products/bash/ar_remote_create.sh:147]: Why has this stuff been gotten rid of sorry? It looks like it's a verification that the remote has been setup?

File: products/bash/release-ar.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+spec:
+  license:
+    accept: true
+  storage:
+    assetDataVolume:
+      class: ibmc-file-gold-gid
+    couchVolume:
+      class: ibmc-block-gold
+  version: 2020.2.1-0
+EOF
```
Comment (James-Kirk) [inline products/bash/release-ar.sh:59]: Is the intention to have all of the products eventual split to their own releases like this?

File: products/bash/ar_remote_create.sh (hunk 1 above)
Comment (DanRoseus) [reply products/bash/ar_remote_create.sh:147]: That's debug code I added in, somehow got on to the operators branch, that's not on master.

File: products/bash/release-ar.sh (hunk 2 above)
Comment (DanRoseus) [reply products/bash/release-ar.sh:59]: I think it makes sense at least initially, should make it easier to install multple instances of a product when we want a dev/test/prod version etc. Originally we were planning to split "release-products.sh" up into separate scripts per product and didn't get around to it.

**Pull request merged**